src/components/MetamaskLogin/template.pug

```diff
@@ -20,17 +20,22 @@ mixin metamask-missing
         )
 
 mixin wrong-chain
-    .icon
-        img(loading="lazy" src="{metamaskImage}" alt="metamask icon")
+    .icon.popup__icon
```

Comment:
@tsejerome how about to put this class here `text-warning`?

Reply:
Text-warning will give it a red colour which was not in the design

Reply:
@tsejerome please check the class values